### PR TITLE
fix: removed password_hash 

### LINKED ISSUE #790: Bug: Remove password hash from being returned on GET /user response
**Describe the bug**
Currently user's password hash is returned on GET /user in the `full_user_api_model`. It should not be returned to frorntend UI since it is a sensitive information. 

**To Reproduce**

1. Login
2. send GET /user request
3. See pasword hash returned as part of user details

**Expected behavior**
The user data returned should not include password hash.

**Screenshots**
Screen Shot 2020-08-31 at 10 24 11 am

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Chrome
 - Version ??

**Additional context**
This is a crross-project issue with BridgeInTech backend. The same issue was also raised on BIT backend repositorry with [issue#102](https://github.com/anitab-org/bridge-in-tech-backend/issues/102)

### PR DESCRIPTION
**Pull request by abhinav72610**
### Description

Removed hash password to be returned on GET /user in the full_user_api_model

Fixes #790

### Type of Change:

- Code

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
This has been tested locally on my device
After changing it looks like
Screenshot_2020-09-02 Mentorship System API

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**

- [x] My changes generate no new warnings 

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by abhinav72610** [comment]
@PrashanthPuneriya can you please review my pr ??

**Review by PrashanthPuneriya — DISMISSED** [review]
@abhinav72610 LGTM :) Congrats on your first contribution to the repo :tada:

**Review by yashasvimisra2798 — DISMISSED** [review]

**Review by vj-codes — DISMISSED** [review]
LGTM 🎉

**Review by SanketDG — DISMISSED** [review]
Looks good to me from a first look but pinging @isabelcosta for a second set of eyes

**Review by isabelcosta — DISMISSED** [review]
Thank you for your contribution @abhinav72610 🎉

**Comment by devkapilbansal** [comment]
@abhinav72610 do we need to remove it from models. Instead, we should not return it using get. I don't know whether that field may be required or not

File: app/api/models/user.py
Comment (isabelcosta) [inline app/api/models/user.py]:
```suggestion
```

**Comment by isabelcosta** [comment]
> @abhinav72610 do we need to remove it from models. Instead, we should not return it using get. I don't know whether that field may be required or not

@abhinav72610 is removing from the model that is used in the GET API response. So it will not be returned, but its still part of the DB model as it should. Does this clarify your questions @devkapilbansal ?

**Comment by isabelcosta** [comment]
The changes made in this PR were tested locally. Following are the results:

1. Code review - Done

2. All possible responses (positive and negative tests) were tested as below:

  * _Get user profile after logging in as Catarina_  
    _Screenshot/gif_:  

BEFORE (on https://mentorship-backend-temp.herokuapp.com/):
image

AFTER (on https://ms-backend-review-pr-819.herokuapp.com/):
image

    _Expected Result_:  Get data as usual, without exposing `password_hash` field
    _Actual Result_: Got user data without `password_hash` field
    
3. Additional testcases covered: N/A 
4. Additional Comments:

5. Status of PR Changed to: Ready to Merge once it has another approval.

**Review by isabelcosta — APPROVED** [review]
Great work @abhinav72610 🎉 
Thank you for your contribution!

**Comment by isabelcosta** [comment]
I tested it (see comment above). I'll merge this, because this already had 4 approvals. The approvals were dismissed after I removed one unnecessary line of code.
cc @rpattath @devkapilbansal

**Pull request merged**
